meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-fio/0006-FIO-internal-io-remove-false-positives-asserts.patch

```diff
@@ -0,0 +1,83 @@
+From f47547b77356d3b3416668fa5bc018ac660ab31b Mon Sep 17 00:00:00 2001
+From: Igor Opaniuk <igor.opaniuk@foundries.io>
+Date: Wed, 31 Aug 2022 10:30:46 +0300
+Subject: [PATCH] [FIO internal] io: remove false positives asserts
+
+Temporary remove falso positives asserts, which are currently being
```

Comment:
Just if you have to fix the typo :) **_s/falso/false/_** **_s/positives/positives/_**